carmajava/launch/drivers.launch

```diff
@@ -27,4 +27,5 @@ If not using simulated drivers they are activated if the respective mock argumen
   <arg name="mock_drivers" default="controller can comms gnss radar lidar camera roadway_sensor" />
   <arg name="vehicle_calibration_dir" default="$(arg vehicle_calibration_dir)"/>
   <!-- File not used in development environment where only mock drivers are used -->
+   <remap from="system_alert" to="/system_alert"/>
```

Comment:
Placing a remap here will not do anything. Remap lines are applied from the provided line down